images/arch/Containerfile

```diff
@@ -12,6 +12,12 @@ COPY extra-packages /
 RUN pacman -Syu --needed --noconfirm - < extra-packages
 RUN rm /extra-packages
 
+# Enable man pages, enable progress bars, enable colors, enable up to 15 parallel downloads
+RUN sed -i -e 's/NoProgressBar/#NoProgressBar/' -e 's/#Color/Color/' -e 's/NoExtract/#NoExtract/' -e 's/ParallelDownloads = 5/ParallelDownloads = 15/' /etc/pacman.conf
```

Comment:
> -e 's/ParallelDownloads = 5/ParallelDownloads = 15/'

I'll gladly ack the man page changes, but this changes lacks and explanation. Generally this is not a good change.

Reply:
Fixed in d7e6849